ping-with-dsl/testplans.ts

```diff
@@ -0,0 +1,66 @@
+import { TestPlans } from "dsl/src/dsl"
+import { run } from "dsl/src/testground-runner"
+import goV0230 from "./go/v0.23.0/image.json"
+import rustV0490 from "./rust/image.0.49.0.json"
+
+type PingParams = {
+    max_latency_ms: number,
+    iterations: number,
+}
+
+const testplans: TestPlans<PingParams> = {
```

Comment:
This is the definition of the test we are running with testground.

Reply:
Do I understand correctly that this definition will be auto-generated?

Reply:
Not this one. This is a minimal example test plan to demonstrate the DSL.

Reply:
When would we run the tests defined here vs the multi-dimensional matrix?

Reply:
It seems to be fairly excessive for an example, in terms of LoC. Did you plan on merging this?

Reply:
Happy to remove this if/when we decide we want to use this DSL for ping tests.